CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.7.4 (2020-09-07)
+
+This maintenance release contains bugfixes since the 1.7.3 release. We deem it high
+priority for upgrading.
```

Comment:
```suggestion
This maintenance release contains bugfixes since the 1.7.3 release. We deem it
high priority for upgrading if TimescaleDB is deployed with replicas (synchronous
or asynchronous).
```